test/testcafe/spec/AuthenticatorYubikeyView_spec.js

```diff
@@ -0,0 +1,107 @@
+import { RequestMock, RequestLogger } from 'testcafe';
```

Comment:
minor nit we usually have enroll and challenge flows tested separately 

eg) 
ChallengeAuthenticatorDuo_spec.js
EnrollAuthenticatorDuo_spec.js

Reply:
The convention I tend to follow (and what appears to be present in this codebase as well) is there is a `_spec` file for every source file. Unlike most of our other authenticators, YubiKey only has one major difference between enroll and verify - the `title` text. To reduce the number of files and duplicate code, we opted for a single View Controller that conditionally renders the title given the scenario:

https://github.com/okta/okta-signin-widget/blob/9172dd832540e3fcf8c6225ab4a7e6290ed18b92/src/v2/view-builder/views/yubikey/AuthenticatorYubiKeyView.js#L12-L15

Given this, I'd prefer to couple them together. Any objection?

Reply:
Well these are e2e test and test file name doesnt have to map to the src file name.  I would slightly prefer it to be isolated tests or have a common file name like YubikeyAuthenticatorEnrollChallenge_spec.js


I am fine as is too.